reviews/models.py

```diff
@@ -127,14 +113,11 @@ def get_continuation_display(self):
         # If this review hasn't concluded, this will only return 'Approved' as
         # this is the default. Thus, we return 'unknown' if we are still pre-
         # conclusion.
-        if self.stage <= Review.COMMISSION:
+        if self.stage <= Review.Stages.COMMISSION:
             return _("Onbekend")
 
         # Get the human readable string
-        continuation = dict(self.CONTINUATIONS).get(
-            self.continuation,
-            self.continuation
-        )
+        continuation = dict(self.Continuations.choices)[self.continuation]
```

Comment:
```suggestion
        continuation = self.Continuations(self.continuation).label
```

This is the proper way to get the label for a choice :)